pkg/costmodel/router.go

```diff
@@ -1482,7 +1482,7 @@ func Initialize(additionalConfigWatchers ...*watcher.ConfigMapWatcher) *Accesses
 
 	var err error
 	if errorReportingEnabled {
-		err = sentry.Init(sentry.ClientOptions{Release: env.GetAppVersion()})
+		err = sentry.Init(sentry.ClientOptions{Release: version.FriendlyVersion()})
```

Comment:
@nealormsbee does our Sentry setup have any particular requirements that this might break?

Reply:
E.g. I'm not sure if `FriendlyVersion` is or is not prefixed with `v`.

Reply:
There's two separate apps for Sentry, one for frontend and backend. We've been reporting backend errors/panics for the last year apparently using the same version `v1.91.0rc-0`. This will just allow us to actually tie a specific version to the error/panic.

Reply:
Definitely won't break the FE project. Should not cause any issue for the BE project either. Sentry will accept with or without the `v` prefix. I learned this the hard way, because the FE was using one format and our script to build sourcemaps was using another, creating two separate releases. One received errors and the other had the sourcemaps :facepalm:

Looks like they don't even have to be semver: https://docs.sentry.io/platforms/javascript/configuration/releases/#bind-the-version

>The value can be arbitrary, but we recommend either of these naming strategies: